Gemfile

```
@@ -38,6 +38,7 @@ group :development, :test do
   gem 'annotate', '~> 3.2', '>= 3.0.3'
   gem 'dotenv-rails', '~> 2.7.6'
   gem 'factory_bot_rails', '~> 5.1', '>= 5.1.1'
+  gem 'parallel_tests', '~> 4.2.1'
```

```suggestion
  gem 'parallel_tests', '~> 4.2'
```

maybe so we allow minor versions upgrades?